### PR TITLE
Add navigation

### LINKED ISSUE #2: [3pts] Add Navigations
In the navigation panel, the user can see links to:
Dress that you selected as a theme.
"Reserve" form.
"My reservations".
"Add the Dress that you selected as a theme" (the link is visible to everybody).
"Delete the Dress that you selected as a theme" (the link is visible to everybody).

### PR DESCRIPTION
**Pull request by brhanuhailu**
# In this branch:
- [ ] added Navigation
- [ ] Modify app 

image

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <li>
+              <Link to="add">Add House</Link>
+            </li>
+            <li>
+              <Link to="delete">Delete House</Link>
+            </li>
+            <li>
+              <Link to="login">sign out</Link>
+            </li>
+          </ul>
```
Comment (Mahamadou-Nouridine) [inline src/components/Navbar.jsx:38]: Kindly remove the default unordered list dots from the menu Items as it makes it different from the base design
Screenshot 2023-09-24 114834

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <>
+      <div className="mobile bars">
+      <span role="button" onClick={() => settray(false)} onKeyDown={() => settray(false)} tabIndex={0}>
+          <i className="fa-solid fa-bars" />
+        </span>
+      </div>
+      <div id="navbar" className={tray ? 'block' : 'show'} role="button" onClick={() => settray(true)} onKeyDown={() => settray(false)} tabIndex={0}>
+        <div className="top">
+          <h2>HOUSE RENT</h2>
+          <ul>
```
Comment (Mahamadou-Nouridine) [inline src/components/Navbar.jsx:19]: [Optional]Kindly add a little bit more spacing between the logo and the navigation Items according to the base design
Screenshot 2023-09-24 115035

**Review by Mahamadou-Nouridine — CHANGES_REQUESTED** [review]
Hey @brhanuhailu, you have done a great job so far, but there is a little required change. Kindly, check the comments below and implement the required changes. You can skip the optional one but it is highly recommended.
68747470733a2f2f6d656469612e74656e6f722e636f6d2f337343486e445f7832524941414141432f616c6d6f73742d74686572652e676966

**Comment by brhanuhailu** [comment]
@Mahamadou-Nouridine  Thank you for your Constructive comments I have made the change. but we have a card for styling just I added a navbar and a little bit of CSS, not a full

**Review by yashodhicy — APPROVED** [review]
STATUS: APPROVED
Hi @brhanuhailu 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! 🎉

TO HIGHLIGHT:

🥇 Your project is professional ✅
🥇Good Job 👍
Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES 🤞 :

**Pull request merged**